application/helpers/expressions/em_manager_helper.php

```diff
@@ -6607,11 +6606,41 @@ function _ValidateQuestion($questionSeq,$force=false)
                     }
                 }
             }
-
+            /**
+             * Control value against value from survey : see #11611
+             */
+            if ($qrel)
```

Comment:
Please don't just insert more code, but break out functions instead and call them.